### DISCUSSION TITLE
源码中的所有引入 vue文件都会显示ts报错找不到，大家会有这样子的问题吗

### DISCUSSION THREAD

**Discussion by badspider7** [element-plus/element-plus#16936] [Q&A]
项目里面的所有引入vue文件都会报这个错误，但是我没改东西，拉下来就这样子了，很早之前是好的，不知道是不是vscode 的插件问题还是啥问题
大家有碰到过这样子的吗
![image](https://github.com/element-plus/element-plus/assets/54093230/831ec242-df54-4f31-a18b-7ca88dfa9b40)

**Comment by warmthsea** [comment]
没有遇到：
你可以检查一下你的 Volar 版本
Typescript 版本
Node 版本
pnpm 版本

**Reply by badspider7** [reply]
好的，
volar是改成 vue-official 插件了，是2.0.19版本
Typescript 版本 4.9.5
Node版本是18.19.1
pnpm版本是 7.3.0

**Reply by warmthsea** [reply]
`npm install pnpm@8.14.1` 升级一下 pnpm 版本

![image](https://github.com/element-plus/element-plus/assets/45450994/8ced0155-d2a9-43fd-b95c-808129be4896)

然后重新执行 `pnpm i`

**Reply by badspider7** [reply]
ok，我去试试看，谢啦

**Reply by badspider7** [reply]
好像还是没有用欸

**Reply by warmthsea** [reply]
检查更新你的 VSCode 版本到最新版

**Reply by warmthsea** [reply]
![image](https://github.com/element-plus/element-plus/assets/45450994/ffce1079-2ef5-4dca-bf1c-1d248a3e12d7)

**Reply by badspider7** [reply]
ok，我去试试看❤️

**Reply by badspider7** [reply]
![image](https://github.com/element-plus/element-plus/assets/54093230/18c9b8a4-0ab8-48b2-9e07-267ac0b38017)
哈哈哈哈这个也是最新的，我估计是vue-official或者是typescript的版本问题啥的
